src/unittest/python/plugins/python/distutils_plugin_tests.py

```diff
@@ -43,6 +44,29 @@ class InstallDependenciesTest(unittest.TestCase):
     def setUp(self):
         self.project = Project(".")
 
+    def test_should_generate_command_abiding_to_configuration(self):
+
+        expected_properties = {
```

Comment:
These are also the default values, so we don't know if the code actually works after this test